.github/workflows/test.yml

```diff
@@ -94,10 +94,6 @@ jobs:
         if: needs.changes.outputs.source == 'true'
         run: make check-events
 
-      - name: Check cargo deny advisories/licenses
-        if: needs.changes.outputs.dependencies == 'true' || needs.changes.outputs.deny == 'true'
```

Comment:
<!-- Thoughts represent an idea that popped up from reviewing. These comments are non-blocking by nature, but they are extremely valuable and can lead to more focused initiatives and mentoring opportunities.  -->
**💭 thought:** ‏I suspect we could remove the `deny` outputs from the Changes workflow as well now.

Reply:
Good thought.